modules/virt-creating-a-primary-udn.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-udn_{context}"]                                  
+= Creating a primary namespace-scoped user-defined network
+
+You can create an isolated primary network in your project namespace. You must use the OVN-Kubernetes layer 2 topology and enable persistent IP address allocation in the user-defined network (UDN) configuration to ensure VM live migration support.
+
+.Prerequisites
+. You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+. Create a `UserDefinedNetwork` object to specify the custom network configuration:
++
+.Example `UserDefinedNetwork` manifest
+[source,yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-l2-net # <1>
+  namespace: my-namespace # <2>
+spec:
+  topology: Layer2 # <3>
+  layer2: 
+    role: Primary # <4>
+    subnets:
+      - "10.0.0.0/24"
+      - "2001:db8::/60" 
+  ipamLifecycle: Persistent # <5>
+----
+<1> The name of the `UserDefinedNetwork` custom resource. 
+<2> The namespace in which the VM is located. This value cannot be `default` or match any global namespaces that are created by the Cluster Network Operator (CNO).
```

Comment:
we need to indicate that this namespace **must** feature the `k8s.ovn.org/primary-user-defined-network`, right @phoracek ?